2022/project_idea.md

```diff
@@ -1,106 +1,116 @@
 # Centralized documentation for computational Bayesian workflow
 
 ## About ArviZ
-ArviZ is an open source project dedicated to building tools for Exploratory Analysis of Bayesian Models.
-Statisticians and data analysts, across both academic and industry, across programming languages
-and across probabilistic programming frameworks use ArviZ to assist them to perform statistical workflows.
-They study a wide range of problems, from the effectiveness of cancer treatments,
-to SpaceX rocket supply chains, to user behavior on the internet.
+ArviZ is an open-source library dedicated to building tools for Exploratory Analysis of Bayesian Models. 
```

Comment:
```suggestion
ArviZ is an open-source project dedicated to building tools for Exploratory Analysis of Bayesian Models. 
```

I would keep project here. ArviZ can mean both project or library (either in python or in julia) but here we are using it as project. The ArviZ project maintains multiple libraries: ArviZ python, ArviZ julia, PSIS.jl, xarray-einstats... and educational resources, all dedicated to Exploratory Analysis of Bayesian Models.

Reply:
Alright, I understand now.